docs/_build/html/nav-tutorial.html

```diff
@@ -410,8 +444,120 @@ <h2>Reproject data onto rectilinear grid<a class="headerlink" href="#Reproject-d
 </div>
 </div>
 <p>For planets with banded structure, it is easy to check the quality of the co-location based on whether or not the bands appear straight in these images. If the centroid is incorrect, then the bands are curved!</p>
-<p>Note here that by default, reproject() will keep the same pixel scale as the original data at the sub-observer point. This means that at every other location, the projected image is oversampled as compared with the original data. This oversampling of the original pixel grid can cause unusual-looking artifacts near the limbs of the planet. To change this default behavior, simply use: reproject(pixscale_arcsec=whatever)</p>
-<p>We plan to support re-projection to arbitrary geometries (e.g., polar) in the near future, so stay tuned.</p>
+<p>Note here that by default, reproject() will keep the same pixel scale as the original data at the sub-observer point. This means that at every other location, the projected image is oversampled as compared with the original data. This oversampling of the original pixel grid can cause unusual-looking artifacts near the limbs of the planet. On the other hand, the nearest-neighbor interpolation used py Cartopy means oversampling might be desired to make things appear smoother. To change this
```

Comment:
Over sampling with nearest neighbor interpolation used in Cartopy can generate smoother results, but also can produce artifacts at the limb of the planet. Caution should be taken in interpreting results.